### PR TITLE
Sep028

### PR DESCRIPTION
**Pull request by nroehner**
Final draft incorporation of SEP 028 into the spec.

### CODE REVIEW

**Comment by palchicz** [comment]
I'll take a look later today.

File: umlet_source/component_instance_2.3.uxf [hunk 1]
```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (palchicz) [inline umlet_source/component_instance_2.3.uxf]: For this umlet source as well as the others, why include the "_2.3" prefix? Can we just call these "component_instance" and let git manage keeping track of the versions?

**Comment by cjmyers** [comment]
I suggested this as it was easier to make a Measured class as an abstract class then have ComponentInstance derive from that in the library implementation.  The bi-product was that Components can now have Measures, which I think is not actually a bad thing.

File: sbol2.tex [hunk 2]
```diff
@@ -65,7 +65,10 @@
...
-sbol:experimentalData,sbol:Experiment,sbol:ExperimentalData,sbol:attachment},
+sbol:experimentalData,sbol:Experiment,sbol:ExperimentalData,sbol:attachment,om:hasNumericalValue,om:symbol,om:alternativeSymbol,om:alternativeLabel,om:longcomment,om:hasFactor,
+om:hasTerm1,om:hasTerm2,om:hasNumerator,om:hasDenominator,om:hasBase,om:hasExponent,om:hasUnit,om:hasPrefix,rdfs:comment,rdfs:label,
+om:Measure,om:Unit,om:SingularUnit,om:CompoundUnit,om:om:UnitMultiplication,om:UnitDivision,om:Exponentiation,om:PrefixedUnit,om:Prefix,om:SIPrefix,
+om:BinaryPrefix,xlmns:om,xlmns:rdfs,xlmns:xsd},
```
Comment (palchicz) [inline sbol2.tex]:
```suggestion
om:BinaryPrefix,xlmns:om,xlmns:rdfs,xlmns:xsd,xml:lang},
```

File: complementary_standards.tex [hunk 3]
```diff
@@ -0,0 +1,1532 @@
...
+    <prov:wasDerivedFrom rdf:resource="http://async.ece.utah.edu/Derivation_example/1"/>
+    <prov:wasDerivedFrom rdf:resource="http://async.ece.utah.edu/Promoter_Derivation/1"/>
+    <prov:wasGeneratedBy rdf:resource="http://async.ece.utah.edu/Derivation_example_GeneratedInstance21_SBOLDesignerActivity/1"/>
+    <prov:wasGeneratedBy rdf:resource="http://async.ece.utah.edu/Derivation_example_SBOLDesignerActivity/1"/>
+    <dcterms:title>Derivation Example</dcterms:title>
+    <dcterms:description>An example derivation</dcterms:description>
+    <sbol:type rdf:resource="http://www.biopax.org/release/biopax-level3.owl#DnaRegion"/>
+    <sbol:role rdf:resource="http://identifiers.org/so/SO:0000804"/>
+    <sbol:component>
+      <sbol:Component rdf:about="http://async.ece.utah.edu/Derivation_example_GeneratedInstance11/Pro_Component/1">
```
Comment (palchicz) [inline complementary_standards.tex]: We can probably delete this dead code

Comment (palchicz) [inline complementary_standards.tex]:
```suggestion
\paragraph{The \sbolheading{hasDenominator} property}\label{sec:hasDenominator}
```

Comment (palchicz) [inline complementary_standards.tex]: deadcode?

Comment (palchicz) [inline complementary_standards.tex]: I'm seeing margin overruns in this section. This applies to the text as well as to the figure.

File: apdx-validation.tex [hunk 4]
```diff
@@ -1126,3 +1146,201 @@ \subsubsection*{Rules for the \class{Experiment} class}
...
+
+\printValid{The \sbolmult{symbol:Unit}{symbol} property of a \sbol{Unit} is REQUIRED and MUST contain a \sbol{String}.\\ Reference: \sec{sec:Unit}}
+
+\printValid{The \sbolmult{alternativeSymbols:Measure}{alternativeSymbols} property of a \sbol{Unit} is OPTIONAL and MAY contain a set of \sbol{String}s.\\ Reference: \sec{sec:Unit}}
+
+\printValid{The \sbolmult{label:Unit}{label} property of a \sbol{Unit} is REQUIRED and MUST contain a \sbol{String}.\\ Reference: \sec{sec:Unit}}
+
+\printValid{The \sbolmult{alternativeLabels:Measure}{alternativeLabels} property of a \sbol{Unit} is OPTIONAL and MAY contain a set of \sbol{String}s.\\ Reference: \sec{sec:Unit}}
+
+\printValid{The \sbolmult{comment:Unit}{comment} property of a \sbol{Unit} is OPTIONAL and MAY contain a \sbol{String}.\\ Reference: \sec{sec:Unit}}
```
[the remark this answers is not in the document]
Comment (nroehner) [reply apdx-validation.tex]: I missed the comment and label properties on those OM classes when I wrote SEP 028.

File: apdx-validation.tex [hunk 5]
```diff
@@ -1126,3 +1146,201 @@ \subsubsection*{Rules for the \class{Experiment} class}
...
+\setcounter{sbolCtr}{14101}
+
+\printValid{A \sbol{Unit} MUST NOT have properties other than the following: \sbol{identity}, \\
+\sbol{persistentIdentity}, \sbol{displayId}, \sbol{version}, \sbol{wasDerivedFroms}, \sbol{wasGeneratedBys}, \sbol{name},\\
+\sbol{description}, \sbol{annotations}, \sbol{attachments}, \sbolmult{symbol:Unit}{symbol}, \sbolmult{alternativeSymbols:Unit}{alternativeSymbols}, \sbolmult{label:Unit}{label},\\
+\sbolmult{alternativeLabels:Unit}{alternativeLabels}, \sbolmult{comment:Unit}{comment}, \sbolmult{longcomment:Unit}{longcomment}, \sbol{hasBase}, and \sbol{hasExponent}.\\ Reference: \sec{sec:UnitExponentiation}}
+
+\printValid{The \sbol{hasBase} property of a \sbol{UnitExponentiation} is REQUIRED and MUST contain a \sbol{URI}.\\ Reference: \sec{sec:UnitExponentiation}}
+
+\printValid{The \sbol{hasExponent} property of a \sbol{UnitExponentiation} is REQUIRED and MUST contain an xsd:integer.\\ Reference: \sec{sec:UnitExponentiation}}
```
[the remark this answers is not in the document]
Comment (nroehner) [reply apdx-validation.tex]: Yes, it should be a integer. This is another place where I mis-translated from the OM to SEP 028.

File: umlet_source/design-build-test.uxf [hunk 6]
```diff
@@ -1,33 +1,35 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<diagram program="umlet" version="14.3.0">
```
Comment (palchicz) [inline umlet_source/design-build-test.uxf]: I think you may have overwritten these changes. Can we revert this?

**Comment by palchicz** [comment]
Closing this for expedience

**Pull request merged**